corehq/apps/hqwebapp/static/hqwebapp/js/components.ko.js

```diff
@@ -1,21 +1,25 @@
 (function () {
+    // standalone components will have their own knockout bindings. Use
```

Comment:
Whether a widget is inside another model isn't really a property of the widget itself. The inline edit widget is used both on its own (for module/form names) and also inside other models (for the releases page and case import history).

You can use the `.ko-template` class to control whether or not bindings are applied - check out https://github.com/dimagi/commcare-hq/commit/3b9ee63d75dfb7ae53bc8db2056d9778baa9af1e